analyzer/src/funTest/assets/projects/external/spdx-tools-python-expected-output.yml

```diff
@@ -135,27 +134,23 @@ packages:
 - id: "PyPI::rdflib:5.0.0"
   purl: "pkg:pypi/rdflib@5.0.0"
   authors:
-  - "Daniel 'eikeon' Krech"
+  - "Daniel 'eikeon' Krech <eikeon@eikeon.com>"
   declared_licenses:
-  - "BSD License"
   - "BSD-3-Clause"
   declared_licenses_processed:
     spdx_expression: "BSD-3-Clause"
-    mapped:
-      BSD License: "BSD-3-Clause"
-  description: "RDFLib is a Python library for working with RDF, a simple yet powerful\
-    \ language for representing information."
+  description: "RDFLib is a Python library for working with"
   homepage_url: "https://github.com/RDFLib/rdflib"
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/d0/6b/6454aa1db753c0f8bc265a5bd5c10b5721a4bb24160fb4faf758cf6be8a1/rdflib-5.0.0-py3-none-any.whl"
+    url: ""
```

Comment:
Odd, why's this missing?

Reply:
See the response on another similar comment.